### PR TITLE
www-client/surf: Rebase 9999 patch.

### PR DESCRIPTION
**Pull request by parazyd**

### CODE REVIEW

File: www-client/surf/surf-2.0.ebuild [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 2020 Gentoo Authors
```
Comment (telans) [inline www-client/surf/surf-2.0.ebuild:1]:
```suggestion
# Copyright 1999-2020 Gentoo Authors
```

File: www-client/surf/surf-9999.ebuild [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 2020 Gentoo Authors
```
Comment (telans) [inline www-client/surf/surf-9999.ebuild:1]:
```suggestion
# Copyright 1999-2020 Gentoo Authors
```

**Comment by telans** [comment]
And you need to sign off using your legal name

**Comment by parazyd** [comment]
Ok, thanks.
I squashed it all now in a single commit.

File: www-client/surf/surf-2.0.ebuild [hunk 3]
```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
```
Comment (juippis) [inline www-client/surf/surf-2.0.ebuild:4]: This requires a revbump and dropping keywords to `~`. Are you sure you want to do this? Nothing wrong with EAPI-6.

File: www-client/surf/metadata.xml [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
-<flag name="savedconfig">Without a saved config.h, this package depends on <pkg>net-misc/curl</pkg> and <pkg>x11-terms/st</pkg> for a default download mechanism</flag>
-</use>
+	<maintainer type="person">
+		<email>parazyd@dyne.org</email>
+		<name>Ivan J.</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy maintainers</name>
+	</maintainer>
```
Comment (juippis) [inline www-client/surf/metadata.xml:11]: Sorry but we won't let you take maintainership of this package before you contribute a while. We see too many people come and dissappear after first commit which causes all kind of bureaucratic work for us.

You can still fix bugs and update this package without being the maintainer for it.

Comment (parazyd) [reply www-client/surf/metadata.xml:11]: While I understand this, I also have to say it's also quite discouraging...

Should I just drop the metadata change then and force push again?

File: www-client/surf/surf-2.0.ebuild (hunk 3 above)
Comment (parazyd) [reply www-client/surf/surf-2.0.ebuild:4]: I see no problem with it.

File: www-client/surf/metadata.xml (hunk 4 above)
Comment (juippis) [reply www-client/surf/metadata.xml:11]: Thanks for understanding. Yes please, do force push.

And as I said, you *can* still contribute and we will process all PRs to maintainer-needed packages. Our trust has been broken too many times so we are on the defensive here. If you stick around we'll welcome you with open arms, but so far I can only find 1 commit from you in 2018.

**Review by juippis — APPROVED** [review]

**Comment by parazyd** [comment]
@juippis What happens now?

**Comment by juippis** [comment]
Now this PR has been merged to our ::gentoo tree. You can find other bugs you want to fix from packages that you use. I see surf has one bug open, 
https://bugs.gentoo.org/742602 
so maybe you can try and fix that bug? Get some practice for Gentoo workflow.

Also try to learn to use git branches, I see you committed to your master branch. You'll most likely get some conflicts when you update upstream master to your local master. So learn git branching when doing PRs.

And do note that this project only gets notified from maintainer-needed PRs, some projects are inactive in Github. Don't be demotivated if some project does not answer in Github, then you'll need to interact with the "old" way using bugzilla, that's still the main official channel to reporting bugs.

**Pull request closed without merging**